website/docs/r/app_service_connection.html.markdown

```diff
@@ -112,6 +112,34 @@ An `authentication` block supports the following:
 
 ---
 
+* `scope` - (Optional) Connection scope in source service. It's namespace name if source is AKS cluster, container name if source is Container app.
+
+* `configuration` - (Optional) The connection information consumed by applications, including secrets, connection strings. A `configuration` block as defined below.
+
+---
+
+A `configuration` block supports the following:
+
+* `action` - (Optional) The action indicates whether to apply configurations on source application. Possible values are `enable` and `optOut`.
+
+* `configuration_store` - (Optional) A `configuration_store` block as defined below.
```

Comment:
We seem to be missing the `configuration_store` block documentation?

Reply:
Will add.